### DISCUSSION TITLE
So, how do i modify my bashrc?

### DISCUSSION THREAD

**Discussion by breningham** [ublue-os/fleek#301] [Q&A]
Hi, Just playing around with fleek, and after installing volta (a js version manager) as a package. it needs to add a folder to the path. normally this is done by running `volta setup` but doing that, it complains that everything is write protected. 

maybe im missing some documentation but i haven't seen anything. 

How do i modify my bash (and i guess, zsh) files with fleek.

**Comment by qnm** [comment]
@breningham This feels like somewhere we need a more general solution as part of fleek itself, tbh.

I'm not sure if you can override the `programs.bash.initExtra` section in Nix without breaking some other part of fleek.

I see `/etc/bashrc` is sourced, perhaps that's a hacky workaround for you by editing it?

https://github.com/ublue-os/fleek/blob/6c181c0e2625239952beb10dda200f6be6c1c3b0/internal/flake/templates/shell.nix.tmpl#L22
